server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminRepairTableIntegrationTestWithDistributedTransactionAdminService.java

```diff
@@ -87,6 +90,14 @@ public void repairTableAndCoordinatorTable_ForTruncatedMetadataTable_ShouldRepai
     super.repairTableAndCoordinatorTable_ForTruncatedMetadataTable_ShouldRepairProperly();
   }
 
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void repairTable_ForCorruptedMetadataTable_ShouldRepairProperly() throws Exception {
```

Comment:
@Torch3333 I think we also need to disable this method if `isExternalServerUsed` is true, right?

Reply:
Yes, thank you for fixing this.